herbert_core/classes/MPIFramework/messages/CompletedMessage.m

```diff
@@ -0,0 +1,21 @@
+classdef CompletedMessage < aMessage
+    % The class the message, indicating the end of a task.
+    %
+    % The message is persistent as should stay in the system until
+    % "ClearAll" signal is received.
+    %
+    %
+    methods
+        function obj = CompletedMessage()
+            % Construct the data message
+            obj = obj@aMessage('completed');
+        end
+    end
+    methods(Static,Access=protected)
+%         function is_pers = get_persist_state()
```

Comment:
disabled code

Reply:
That's a difficult question. I have removed special status from "completed" message. Its now like any other generic messages, so class should probably be removed. But removing the class is abit of a pain as it is in many places (should be instatiated through factory)  It probably worth a small ticket, so feel free to give it to me if you think it is important